README.md

```diff
@@ -482,7 +482,7 @@ There are many ways to set up your local Python environment. We suggest some san
 To build and install pydgraph locally, run
 
 ```sh
-pip install -e .[dev]
+pip install -e ".[dev]"
```

Comment:
why the quotes?

Reply:
zsh interprets it different and complains.

Reply:
:+1: